exercises/hello_rubyric/config.yaml

```diff
@@ -0,0 +1,17 @@
+
+_external: true
```

Comment:
Why was this internal value required?

Reply:
That is a good question. I was actually taking some config values from the [o1 course](https://version.aalto.fi/gitlab/jsorva/o1matsku/-/blob/2c0046138259b14639b563e70575b018c984c13f/exercises/ex_own_adventure/config_en.yaml)

Reply:
Most likely the parameter `_external` is not necessary in this case. Therefore, I will remove it.  @Mankro which group size setting?

Reply:
Actually, you can see in `toc_config.py` that the `_external` field is used in an if branch. Thus, it can't be removed.

The group size settings consist of `min_group_size` and `max_group_size`. They haven't been added to the submit RST directive.

Reply:
![image](https://user-images.githubusercontent.com/10437475/94557047-b6b27300-0266-11eb-8218-78aedb6c4c07.png)

Reply:
That is what I said. Nobody has added the group size settings to the submit RST directive. They are missing. Adding them is a task for another pull request. Currently, one needs to use a config.yaml file in order to define a group exercise.

Reply:
https://github.com/apluslms/a-plus-rst-tools/issues/70

Reply:
When I said that the group size settings are missing from the RST submit directive, I meant that they are a missing feature and not YET implemented in the code. You seem to have read it as "missing from the RST example, let's add the options to the example exercise". Now it should be clear!

Reply:
Yes, now it is clear.